src/groups/mqb/mqbblp/mqbblp_queuehandle.cpp

```diff
@@ -610,12 +606,14 @@ void QueueHandle::registerSubscription(unsigned int downstreamSubId,
                                        const bmqp_ctrlmsg::ConsumerInfo& ci,
                                        unsigned int upstreamId)
 {
-    if (d_throttledSubscriptionInfo.requestPermission()) {
-        BALL_LOG_INFO << "QueueHandle [" << this
-                      << "] registering Subscription [id = " << downstreamId
-                      << ", downstreamSubQueueId = " << downstreamSubId
-                      << ", upstreamId = " << upstreamId << "]";
-    }
+    static const int                k_MAX_INSTANT_MESSAGES = 10;
```

Comment:
Maybe, move `k_MAX_INSTANT_MESSAGES` and `k_NS_PER_MESSAGE` out in a single place?  They are the same for all occurrences most likely.